src/EditorFeatures/CSharpTest/CodeActions/ConvertLocalFunctionToMethod/ConvertLocalFunctionToMethodTests.cs

```diff
@@ -550,6 +550,54 @@ void M()
             return null;
         }}
     }}
+}}");
+            }
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertLocalFunctionToMethod)]
+        public async Task TestWholeMethodSelectionPositon()
+        {
+            await TestAsync("[|C LocalFunction(C c)");
```

Comment:
Probably best to split this into two separate tests, and you can avoid local functions and string interpolations to improve readability.

Reply:
Sure, will fix.